modules/system-management/pages/system-metrics.adoc

```diff
@@ -0,0 +1,102 @@
+= Metrics Reporting
+:description: Overview of metrics gathering in TigerGraph Server.
+
+You can use `gadmin metric` commands to collect the following metrics related to state of resource usage on your instances:
+
+* CPU usage
+* Memory usage
+* Disk space usage
+* Network-related information
+
+For a full list of options available to `gadmin metric`, run `gadmin metric -h` in the command line to view the help text for the command.
```

Comment:
We should say this, however, the wording makes it seem as though we are not listing the full list of metrics available. A complete list needs to be provided in the documentation

Reply:
Linked to the reference for the command